tensorflow_probability/python/sts/decomposition.py

```diff
@@ -39,7 +39,7 @@ def _split_covariance_into_marginals(covariance, block_sizes):
 
 
 def _decompose_from_posterior_marginals(
-    model, posterior_means, posterior_covs, parameter_samples):
+    model, posterior_means, posterior_covs, parameter_samples, initial_step=0):
```

Comment:
Would you mind adding an entry for `initial_step` to the docstring as well? For example:

```initial_step: optional `int` specifying the initial timestep of the decomposition.```

Reply:
I added an entry to the docstring.